src/containers/Tablet/Tablet.tsx

```diff
@@ -48,6 +48,12 @@ export const Tablet = () => {
         error,
     } = useTypedSelector((state) => state.tablet);
 
+    useEffect(() => {
```

Comment:
Does it work properly with StrictMode (double component mount)?

Reply:
We are on react 17, there is no double mount in strict mode here

Reply:
It might be problems in this place when we decide to migrate to React 18. Lets mark it not to forget to pay attention to this piece of code.